vitrina/datasets/models.py

```diff
@@ -96,6 +96,7 @@ class Meta:
         managed = False
         db_table = 'dataset'
         unique_together = (('internal_id', 'organization_id'),)
+        ordering = ('-published',)
```

Comment:
Usually it is a bad idea to add ordering at this low level, because it interferes with other queries, especially aggregations, which can end up in unexpected query results.

https://docs.djangoproject.com/en/3.2/topics/db/aggregation/#interaction-with-order-by

Please move this to a view.